src/basic/copy.h

```diff
@@ -15,6 +15,7 @@ typedef enum CopyFlags {
         COPY_SAME_MOUNT  = 1 << 3, /* Don't descend recursively into other file systems, across mount point boundaries */
         COPY_MERGE_EMPTY = 1 << 4, /* Merge an existing, empty directory with our new tree to copy */
         COPY_CRTIME      = 1 << 5, /* Generate a user.crtime_usec xattr off the source crtime if there is one, on copying */
+        COPY_SIGINT      = 1 << 6, /* Check for SIGINT regularly and return EINTR if seen (caller needs to block SIGINT) */
```

Comment:
"regularly" is not really true. We may be copying a single file for an hour or two. It'd be more straightforward to say "after copying each file".

Reply:
we are also querying for SIGINT while copying large files

Reply:
We do? I thought we generally call `copy_file_range(..., SSIZE_T_MAX)`.

Reply:
hmm, true, if that works we'll indeed not check again